drivers/misc/CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ endif()
 
 if(NOT CONFIG_DEV_OPTEE_NONE)
   list(APPEND SRCS optee.c)
+  list(APPEND SRCS optee_rpmsg.c)
```

Comment:
let's call the backend optee_socket.c? since it supports both rpmsg and local socket.